package.json

```diff
@@ -21,7 +21,7 @@
     "dotenv": "^8.2.0",
     "ethereum-blockies-base64": "^1.0.2",
     "framer-motion": "^4.1.3",
-    "gatsby": "^4.10.0",
+    "gatsby": "^4.14.0",
```

Comment:
Bumped the version to be able to use the flag `GRAPHQL_TYPEGEN`.

Reply:
`GRAPHQL_TYPEGEN` was experimental. Bumped again to v4.15.0 to use the now stable `graphqlTypegen` feature.